README.md

```diff
@@ -0,0 +1,34 @@
+# BETTER BUILDS
+This React website is for users to submit and review PC builds that will be posted to PostgreSQL database. Users can also edit their reviews and vote on the reviews of other users with Express backend, running on a NodeJS environment.
+## Contributors: 
+Nick DeRis - github.com/dericholas
+Nick Kwiatkowski - github.com/nickkwiat
+Robert Le - github.com/fuzzyphantasm
+Corey Pierce - github.com/cornfrog
+Marlon Sanchez - github.com/MarlonS419 
+## Installation
+```
+git clone https://github.com/MarlonS419/better-builds
+```
+then Navigate to <localhost:3000/> in your browser
```

Comment:
users will need to first go through the installation below before they can navigate to the page using localhost - let's swap and move this after installation